### PR TITLE
data:PushkarRoy

### PR DESCRIPTION
**Pull request by roy2pushkar**

### CODE REVIEW

File: PushkarRoy.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Pushkar Roy",
+  "type": "personal",
+  "bio": "Software Developer **Full Stack developer**. Travel enthusiast ",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/roy2pushkar",
+      "icon": "FaTwitter"
+    }
```
Comment (shyamtawli) [inline PushkarRoy.json:10]: add `,` after `}`

Comment (vedprakashnautiyal) [inline PushkarRoy.json:10]:
```suggestion
    },
```

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]
Also please add your json file in data folder.......

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey Pushkar, please move your file into data directory and [rename it here](https://github.com/roy2pushkar/LinkFree/edit/roy2pushkar-patch-1/PushkarRoy.json) to exact same as your username. 

Change `PushkarRoy.json`  -->  `data/roy2pushkar.json`

Also, make the changes requested by Shyam and Vedprakash above. 🛠

**Comment by roy2pushkar** [comment]
please help...why my JSON check failed ??

**Comment by SaraJaoude** [comment]
I am going to close this Pull Request as this is a duplicate of #5999

**Pull request closed without merging**